### PR TITLE
Updating SNP ID

### PR DESCRIPTION
**Pull request by rnmitchell**
For UAS v2.5, one of the SNP IDs was changed from ```N29insA``` to ```rs312262906```.  Updating the various IDs for this SNP to be consistent (STRait Razor outputted the SNP as ```rs312262906_N29insA```).

### CODE REVIEW

**Comment by rnmitchell** [comment]
This is ready for review @standage

File: lusSTR/wrappers/snps_format.py [hunk 1]
```diff
@@ -154,6 +155,8 @@ def process_sigprep_snps(foren_df, sampid, projid, analyid):
     data_list = []
     for j, row in foren_df.iterrows():
         snpid = foren_df.loc[j, "Locus"]
+        # if snpid == "N29insA" or snpid == "rs312262906_N29insA":
+        #    snpid = "rs312262906"
```
Comment (standage) [inline lusSTR/wrappers/snps_format.py:159]: Did you want to keep this commented-out code?

**Review by standage — COMMENTED** [review]
Delightfully straightforward changes! Just one question: happy to approve and merge after that.

File: lusSTR/wrappers/snps_format.py (hunk 1 above)
Comment (rnmitchell) [reply lusSTR/wrappers/snps_format.py]: Nope- good catch. Will remove!

**Comment by rnmitchell** [comment]
Ok- now ready for you @standage

**Review by standage — APPROVED** [review]

**Pull request merged**